RFCs/polka-index.md

```diff
@@ -0,0 +1,79 @@
+# RFC-001: Polka-index
```

Comment:
Shall we just call it Indexer/Storage Indexer/Polka Storage Indexer? And let's use the name consistently (on the diagrams, with capitalization and all). 

Why? `Polka-index` makes it feel bigger than it is.

Reply:
I am okay with that, polka-index doesn't feel like a good name to me either. I prefer `Indexer` or `Storage Indexer`

Reply:
`Storage Indexer` is half a misnomer, since you're not indexing the storage itself, but rather the Storage Providers.

`Indexer` is too general.

`Storage Network Indexer`, `Provider Network Indexer`, or similar would be better, bonus points if you can come up with an acronym that is also a name 😅 

Either way, `Polka Index` was the name picked for us, we MUST make it clear that it changed